source/releasenotes/compat.rst

```diff
@@ -20,35 +20,32 @@ Supported OS Versions for Management Server
 -------------------------------------------
 
 This section lists the operating systems that are supported for running
-CloudStack Management Server. Note that specific versions of the
-operating systems are tested, so compatibility with CentOS 6.3 may not
-indicate compatibility with CentOS 6.2, 6.1 and so on.
+CloudStack Management Server. .
 
--  RHEL versions 6.3, 6.5, 6.6 and 7.0
--  CentOS versions 6.8, 7
+-  RHEL versions 7.x
```

Comment:
7.x is verified for all .x?

Reply:
As verified as the previosly stated version "7".. joke aside, we've been updating templates in Trillian many times, so, implicitly, yes, all version 7.x so far